build/bin.LICENSE

```diff
@@ -1597,3 +1597,642 @@ licenses.
+    DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+    SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+    OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+    OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+- com.google.protobuf.protobuf-java-<VERSION>.jar in omega-edit-scala-server-<VERSION>.zip
+  This product bundles "Protocol Buffers" from the above files.
+  These files are are avaiable under the License:
+    Copyright 2008 Google Inc.  All rights reserved.
+
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are
+    met:
+
+        * Redistributions of source code must retain the above copyright
+    notice, this list of conditions and the following disclaimer.
+        * Redistributions in binary form must reproduce the above
+    copyright notice, this list of conditions and the following disclaimer
+    in the documentation and/or other materials provided with the
+    distribution.
+        * Neither the name of Google Inc. nor the names of its
+    contributors may be used to endorse or promote products derived from
+    this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+    A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+    OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+    SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+    LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+    DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+    THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+    (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+    OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+    Code generated by the Protocol Buffer compiler is owned by the owner
+    of the input file used when generating it.  This code is not
+    standalone and requires a support library to be linked with it.  This
+    support library is itself covered by the above license.
+
+- org.checkerframework.checker-compat-qual-<VERSION>.jar in omega-edit-scala-server-<VERSION>.zip
+  This product bundles "Checker Framework" from the above files.
+  These files are are avaiable under the License:
+    The Checker Framework
+    Copyright 2004-present by the Checker Framework developers
+
+
+    Most of the Checker Framework is licensed under the GNU General Public
+    License, version 2 (GPL2), with the classpath exception.  The text of this
```

Comment:
GPL is not compatible with ALv2 and is deemed a [Category-X](https://www.apache.org/legal/resolved.html#category-x) license by ASF. Any non-optional GPL dependencies must be removed or be optional before this change can be accepted.

Reply:
That's a rough one because this is required by gRPC for the Scala server

Reply:
This ones License was also weird because it had GPL but it also had MIT? If it helps this where it came from https://github.com/typetools/checker-framework/blob/master/LICENSE.txt

Reply:
Do you know of way to remove a dependency's dependency?

Reply:
You can exclude transitive dependencies in sbt:

https://www.scala-sbt.org/1.x/docs/Library-Management.html#Exclude+Transitive+Dependencies

But that only works if those transitive dependencies aren't actually needed. It's pretty likely they are needed, and you'll end up getting runtime exceptions once something tries to use it.

The only way around this is code changes to the dependency, or generally switching to a different dependency that provides similar functionality.

Reply:
Yeah I am currently testing this on the omega-edit side to see if I find any issues with the currently implemented items

Reply:
I was able to successfully remove it and the server works the exact same so for some reason that have that has a dependency and don't use it. Believe it or not I found a couple different things that did this for some reason so I will remove that from bin.LICENSE and release a new omega-edit version.

Reply:
@stevedlawrence this license was removed from bin.LICENSE everything should be good now